cmgd/cmgd_trxn.c

```diff
@@ -1171,7 +1171,8 @@ static int cmgd_trxn_prepare_config(cmgd_trxn_ctxt_t *trxn)
 	nb_ctxt.user = (void *)trxn;
 	ret = nb_candidate_validate_yang(nb_config, err_buf, sizeof(err_buf)-1);
 	if (ret != NB_OK) {
-		(void) cmgd_trxn_send_commit_cfg_reply(trxn, false, err_buf);
+		(void) cmgd_trxn_send_commit_cfg_reply(trxn, false,
```

Comment:
Why can't we return err_buf directly?

Reply:
In Case of validation failure, err_buf is NULL

Reply:
err_buf cannot be NULL. And in case of failure err_buf should NOT be empty too.